src/main/java/org/jenkinsci/test/acceptance/po/WizardCreateAdminUser.java

```diff
@@ -65,14 +65,10 @@ public WizardCreateAdminUser createAdminUser(String userName, String password, S
         }
 
         driver.switchTo().defaultContent();
-        control(by.css(".btn-primary.save-first-user")).click();
+        control(by.css(".btn-primary.save-first-user")).clickAndWaitToBecomeStale(Duration.ofSeconds(time.seconds(5)));
         return this;
     }
 
-    public void shouldCreateTheUserSuccessfully() {
-        waitFor(hasContent("Jenkins is ready!"));
-    }
-
```

Comment:
1. Jenkins does not show the "Jenkins is ready!" message after creation of the admin user, it shows the set URL dialog!
2. this worked because whilst you may think this is waiting for Jenkins to have that message, it is actually not waiting as there is no `until` and this is not using the `waitFor(Matcher, timeout)` method.  ie it is a complete no-op as far as checking anything is concerned.